src/content/structured/styles/components/IconFinder/index.css

```diff
@@ -20,7 +20,7 @@
 
 .icon-group-links:first-child {
   color: var(--ic-color-secondary-text);
-  margin-top: var(--ic-space-md);
+  padding-top: calc(var(--ic-space-md) + var(--ic-space-sm));
```

Comment:
I think it may be worth adding this calculated padding-top to the .icon-group-links parent element above rather than to  the first-child. This would clean up code by only calculating padding for the element in one place.